tensorflow/lite/tools/evaluation/tasks/README.md

```diff
@@ -0,0 +1,21 @@
+# TFLite Model Task Evaluation
+
+This page describes how you can check the accuracy of quantized models to verify that any degradation in accuracy is within acceptable limits.
+
+## Tools
+There are three different binaries which are supported. A brief description of each is provided below.
+
+### Inference Diff Tool
+This binary compares TensorFlow Lite execution in single-threaded CPU inference and user-defined inference.
+
+### Image Classification Evaluation
+This binary evaluates TensorFlow Lite models trained for the [ILSVRC 2012 image classification task.](http://www.image-net.org/challenges/LSVRC/2012/)
+
+### Object Detection Evaluation
+This binary evaluates TensorFlow Lite models trained for the bounding box-based [COCO Object Detection](https://cocodataset.org/#detection-eval) task.
+
+For a more detailed specification of each tool refer to:
```

Comment:
Consider merging each item to the sections above accordingly?

Again, I think it's also worth referring to "https://www.tensorflow.org/lite/performance/delegates#accuracy_correctness" at the end.